### DISCUSSION TITLE
difference between buttons (digital, analog)

### DISCUSSION THREAD

**Discussion by kimgalibert** [shanteacontrols/OpenDeck#295] [General]
is there any difference between a button connected to a digital input vs one connected to analog?

for instance, any limits on the digital vs the analog?  wire length perhaps?

more specifically, for a sustain pedal, is there a benefit to using analog?

thanks!

**Comment by paradajz** [comment]
For buttons there isn't any difference really. The only difference is that you cannot connect encoders to analog inputs. Re: Wire length - have no idea actually, it shouldn't really matter.
